File: SCALA/Cap05/c05ex11.scala
// c05ex11.scala

import scala.io.StdIn._

object Main {
  def main(args: Array[String]): Unit = {
    var I: Int = 1
    var N: Int = 0
    var R: Int = 0

    for (I <- 1 to 5 by 1) {
      print("Entre um valor numerico: ")
      N = readInt()

      R = N * 3

      println("Resultado = " + R + "\n")
    }

    print("\nTecle <Enter> para encerrar... ")
    readLine()
  }
}

File: KOTLIN/Cap05/c05ex11.kt
// c05ex11.kt

fun main() {
    var N: Int
    var R: Int	

    for (I in 1..5 step 1) {
        print("Entre um valor numerico: ")
        N = readLine()?.toInt() ?: 0

        R = N * 3

        println("Resultado = $R\n")
    }

    print("\nTecle <Enter> para encerrar... ")
    readLine()
}
